packages/@react-aria/interactions/src/useMove.ts

```diff
@@ -223,7 +228,7 @@ export function useMove(props: MoveEvents): MoveResult {
     };
 
     return moveProps;
-  }, [state, onMoveStart, onMove, onMoveEnd, addGlobalListener, removeGlobalListener]);
+  }, [onMoveStart, onMove, onMoveEnd, addGlobalListener, removeGlobalListener]);
```

Comment:
i don't think state was hurting anything to be included in this array, it's a ref and shouldn't ever change, why was it removed?